workers_kv.go

```diff
@@ -83,18 +84,53 @@ func (api *API) CreateWorkersKVNamespace(ctx context.Context, req *WorkersKVName
 //
 // API reference: https://api.cloudflare.com/#workers-kv-namespace-list-namespaces
 func (api *API) ListWorkersKVNamespaces(ctx context.Context) (ListWorkersKVNamespacesResponse, error) {
-	uri := fmt.Sprintf("/accounts/%s/storage/kv/namespaces", api.AccountID)
+	uri := fmt.Sprintf("/accounts/%s/storage/kv/namespaces?per_page=100", api.AccountID)
```

Comment:
Per #377, increasing this to reduce the number of API calls made.

Reply:
I'm fine with this for now but if we get any more query parameters, we are going to need to swap to using `url.Values` to ensure we don't thrash the others.